style/ruby/.rubocop.yml

```diff
@@ -1,7 +1,7 @@
 # https://raw.githubusercontent.com/thoughtbot/hound/master/config/style_guides/ruby.yml
 
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.5.8
```

Comment:
I believe you need to drop the `.8` here based on the documentation
https://docs.rubocop.org/rubocop/1.0/configuration.html#setting-the-target-ruby-version

Reply:
I've run with 2.5.8 locally.  let me give 2.5 a try.

Reply:
@ghouston Also from the docs it looks like Rubocop can simply use our `.ruby-version` file so we don't have to keep it in sync in two places:

> ... RuboCop will then check your project for .ruby-version and use the version specified by it.

Reply:
2.5 is working just as well.  i checked the warnings that 2.3 gave and they have gone away with 2.5.  update pushed.

Reply:
Nevermind what I said, I just realized this isn't the Archimedes repo 😅